### PR TITLE
Check path and execute permissions

### PR DESCRIPTION
**Pull request by adrianwong**

### CODE REVIEW

**Comment by alfiedotwtf** [comment]
LGTM

**Review by mikeday — APPROVED** [review]
Looks good to me!

File: src/Prince.php [hunk 1]
```diff
@@ -149,6 +149,13 @@ class Prince
      */
```
Comment (wezm) [inline src/Prince.php:149]: Suggest adding some details to this comment (which I assume is where the [API docs](https://yeslogic.github.io/prince-php-wrapper/current/classes/Prince-Prince.html#method___construct) come from) describing the scenarios in which exceptions will be thrown.

Comment (adrianwong) [reply src/Prince.php:149]: Good idea, thanks.

**Comment by adrianwong** [comment]
Note: swapped `file_exists` for `is_file` as the latter additionally checks if the supplied path is a directory.

**Pull request merged**